packages/utils/cli/src/scan/loadConfig.mjs

```diff
@@ -0,0 +1,32 @@
+import { existsSync } from 'fs'
+import merge from 'lodash.merge'
+
+import * as defaultConfig from './config.mjs'
+
+export async function loadConfig(configFileRoute, { logger }) {
+  try {
+    if (existsSync(configFileRoute)) {
+      logger.info('ℹ️ Loading spark-ui custom configuration file')
+      const { default: customConfig } = await import(configFileRoute)
+
+      const config = {
+        adoption: merge(defaultConfig, {
+          ...customConfig.adoption,
+          imports: customConfig.imports || defaultConfig.imports,
+          extensions: customConfig.extensions || defaultConfig.extensions,
+        }),
+      }
+
+      return config
+    } else {
+      logger.warning('⚠️ No custom configuration file found')
+      logger.info('ℹ️ Loading default configuration')
+
+      return { ...defaultConfig }
+    }
+  } catch (error) {
+    logger.error('💥 Something went wrong loading the custom configuration file')
+
+    return { ...defaultConfig }
```

Comment:
Don't you think it would be clearer for CLI user to throw an error and exit here?
For now we have no details on what was the failure, and as we fallback to default config, user may not notice his config is wrong :thinking:

Reply:
@andresin87 If I'm not wrong, we still return a config on `loadConfig` failure? As said, don't you think we should rather throw error and exit the process, without returning a value?
Actually if would be difficult for the user to identify that he/she defined an invalid configuration file, if the process still goes on. He/she would end with results but not expected ones :/

Reply:
@andresin87 We still don't kill the process here, and return a config, which still could be confusing... That said, I'm not sure we really need this `try...catch` as `path.join(process.cwd(), configuration || '.spark-ui.cjs')` shouldn't throw any error, wdyt? cc @acd02 (other eyes may be interesting here)

Reply:
I must admit that just by looking at the code like this, I don’t think I’ll have anything relevant to add 😅 

I would have to checkout your branch and throw some `console.log` around to get a better gist at what exactly this code is doing